src/OrchardCore/OrchardCore.Infrastructure.Abstractions/Entities/ICacheableEntityExtensions.cs

```diff
@@ -0,0 +1,27 @@
+namespace OrchardCore.Entities;
+
+public static class ICacheableEntityExtensions
+{
+    public static bool TryGetValue(this ICacheableEntity cache, string key, out object value)
```

Comment:
Check the `key` if it's not `null` or empty

Reply:
Not needed as it is handled by the `Get` method